rq/job.py

```diff
@@ -393,6 +398,35 @@ def dependents_key(self):
         """The Redis key that is used to store job dependents hash under."""
         return self.dependents_key_for(self.id)
 
+    @property
+    def dependencies_key(self):
+        return '{0}{1}:dependencies'.format(self.redis_job_namespace_prefix, self.id)
+
+    def fetch_dependencies(self, watch=False, pipeline=None):
+        """
+        Fetch all of a job's dependencies. If a pipeline is supplied, and
+        watch is true, then set WATCH on all the keys of all dependencies.
+
+        Returned jobs will use self's connection, not the pipeline supplied.
+        """
+        if pipeline is None:
+            pipeline = pipeline or self.connection
+
+        if watch and self._dependency_ids:
+            pipeline.watch(*self._dependency_ids)
+
+        jobs = self.fetch_many(
```

Comment:
Nitpick: these three lines is short enough, can you please make this into one liner?

Reply:
In 7c690087a9f4d287d400e8b489eb851e94fedfc0:
https://github.com/rq/rq/blob/7c690087a9f4d287d400e8b489eb851e94fedfc0/rq/job.py#L418